framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/DeploymentScenarioBuilderFactory.java

```diff
@@ -35,6 +36,7 @@
     WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder getWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder();
     KieServerWithExternalDatabaseScenarioBuilder getKieServerWithExternalDatabaseScenarioBuilder();
     KieServerWithDatabaseScenarioBuilder getKieServerWithDatabaseScenarioBuilder();
+    ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder getClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder();
```

Comment:
Just to clarify: what does the work `Persistent` in scenario mean? Doesn't presence of `Database` already imply persistence?

Reply:
@jhrcek Not necessarily. Persistent means that the deployment state is persisted through pod restarts. Theoretically database doesn't imply persistence as there may be ephemeral databases (though not used in our templates).